### ISSUE TITLE
gcmd.sciencekeywords fail to load data into editor

### ISSUE THREAD

**Issue by cnrdh** [npolar/formula#5]
https://app2.data.npolar.no/dataset/8baab493-7a95-506f-a504-55d424faab45/edit
=> 0 GCMD sciencekeywords

See https://api.npolar.no/dataset/8baab493-7a95-506f-a504-55d424faab45

``` json
"gcmd": {
"sciencekeywords": [
{
"Category": "EARTH SCIENCE",
"Topic": "HUMAN DIMENSIONS",
"Term": "ENVIRONMENTAL IMPACTS",
"Variable_Level_1": "CONTAMINANTS"
},
{
"Category": "EARTH SCIENCE",
"Topic": "OCEANS",
"Term": "OCEAN CHEMISTRY",
"Variable_Level_1": "STABLE ISOTOPES"
}
]
```

**Comment by cnrdh** [comment]
Culled the schema for all but gcmd.sciencekeywords - and the bug remains

**Comment by cnrdh** [comment]
Possibly related issue with string enum roles nested into array of people

**Comment by rexso** [comment]
The problem was caused by the material design template, due to lack of multi-select support.
I have pushed a commit to npdc-material, fixing the problem: npolar/npdc-material@d0dedef

**Comment by rexso** [comment]
Was actually unrelated, but has been fixed as of 52beae3c6eac2c82461298614ea047cb13201699.
